tardis/montecarlo/montecarlo_numba/interaction.py

```diff
@@ -56,10 +54,13 @@ def thomson_scatter(r_packet, time_explosion):
 def line_scatter(r_packet, time_explosion, line_interaction_type, numba_plasma):
     """
     Line scatter function that handles the scattering itself, including new angle drawn, and calculating nu out using macro atom
-    r_packet: RPacket
-    time_explosion: float
-    line_interaction_type: enum
-    numba_plasma: NumbaPlasma
+    
+    Parameters
+    ----------
+    r_packet : RPacket
+    time_explosion : float
+    line_interaction_type : enum
+    numba_plasma : NumbaPlasma
```

Comment:
Same goes here - find out the entire path of `NumbaPlasma` class. It should be something like `tardis. ... .NumbaPlasma`

Why: unlike default types like int, float, etc., these are objects. So when you check output of `type(<object of interest>)` in a python kernel, you'll get the class from which object was instantiated which looks like `<package>.<module>.<sub-module>. ... .<class>`. Hope this makes things clearer!